src/it/unimi/dsi/fastutil/io/FastBufferedOutputStream.java

```diff
@@ -125,6 +125,8 @@ public FastBufferedOutputStream(final OutputStream os) {
 	}
 
 	private void dumpBuffer(final boolean ifFull) throws IOException {
+		if (pos == 0)
+			return; // nothing to dump
```

Comment:
Braces are good even if it's just one line.

https://embeddedgurus.com/barr-code/2014/03/apples-gotofail-ssl-security-bug-was-easily-preventable/

Reply:
Fair enough. I normally don't add braces to simple early returns and other typical one-line cases (judging by how typical a case is rather than by the actual number of lines), and that code is already full of single-line `if`s, so I thought it was OK... But certainly more braces won't hurt! Fixed.

Reply:
first and foremost it should be consistent with the style guide of the project. If one-line-returns don't use braces, don't start adding this - it causes way more confusion.

btw: using a code-formatter (correcting the indent) "prevents" the goto-fail in the same way as adding braces :-) the problem there is not the braces in particular but breaking with the code style / not properly indicating the end of a block. If braces were essential, python code would arguably be inherently unsafe ;-)

Reply:
This particular project doesn't seem to have a style guide, though.

Looking at the rest of the code, not only it seems not to use braces, it also seems to place both `if` and its body on the same line, which I'm not a fan of, but I guess following the style is probably the best choice. And besides, in case of a single `return` at the start of the function, it doesn't really matter much.

Re-pushed again, this time following the style of the rest of the files.